process/interface.go

```diff
@@ -1113,7 +1113,10 @@ type ScheduledTxsExecutionHandler interface {
 	Execute(txHash []byte) error
 	ExecuteAll(haveTime func() time.Duration) error
 	GetScheduledSCRs() map[block.Type][]data.TransactionHandler
-	SetScheduledSCRs(mapSCRs map[block.Type][]data.TransactionHandler)
+	SetScheduledRootHashAndSCRs(rootHash []byte, mapSCRs map[block.Type][]data.TransactionHandler)
+	GetScheduledRootHashForHeader(headerHash []byte) ([]byte, error)
+	RollBackToBlock(headerHash []byte) error
+	SaveState(headerHash []byte)
 	GetScheduledRootHash() []byte
```

Comment:
Are GetScheduledRootHash and SetScheduledRootHash still used outside the component? If not we can remove them from interface and made them private there.

Reply:
yes, still used from baseProcess and baseSync